cosmotheka/tests/test_mapper_p18cmbk.py

```diff
@@ -55,7 +56,7 @@ def test_get_signal_map():
     d = m.get_signal_map()
     assert len(d) == 1
     d = d[0]
-    assert np.all(np.fabs(d-1) < 0.02)
+    assert np.all(np.fabs(d) < 0.02)
```

Comment:
what has changed? why did `d-1` work before and now you use `d`?

Reply:
The alms are generated from a constant map=1, so the only non-zero alm is a00. Now we're setting that to zero (as we should), so here we're checking that the recovered map is 0 everywhere.